### PR TITLE
JetBrains: Update README

### PR DESCRIPTION
**Pull request by iskyOS**
Updated the README (similar to the VS Code page) with latest messaging.

- Updated feature messaging
- Updated feature ordering
- Added info on the code graph
- Added language support info
- Added enterprise support info

## Test plan

README updates only.

### CODE REVIEW

File: client/jetbrains/README.md [hunk 1]
```diff
@@ -2,40 +2,39 @@
...
+Cody understands your entire codebase — not just your open files. Ask Cody any question about your code, and it will use Sourcegraph's code graph to answer using knowledge of your codebase.
 
-### 🔨 Let Cody write code for you
+For example, you can ask Cody:
 
-Cody can provide real-time code autocompletions as you're typing. As you start coding, or after you type a comment, Cody will look at the context around your open files, your file history, and your entire codebase to predict what you're trying to implement and provide suggestions.
+- "How is our app's secret storage implemented on Linux?"
+- "Where is the CI config for the web integration tests?"
+- "Write a new GraphQL resolver for the AuditLog"
+- "Why is the USerConnectionResolver giving an "unknown user" error, and how do I fix it?"
```
Comment (taylorsperry) [inline client/jetbrains/README.md:29]: Should `USerConnectionResolver` be `UserConnectionResolver` (lc "s")?

**Review by taylorsperry — APPROVED** [review]
So much better, thank you! Caught what I think is a small typo, but otherwise looks great. One thing I noticed: the screenshots still have the "Recipes" tab. We should swap those out once the new release is cut (probably middle of next week).

File: client/jetbrains/README.md (hunk 1 above)
Comment (iskyOS) [reply client/jetbrains/README.md:29]: Yes! Thank you.
```suggestion
- "Why is the UserConnectionResolver giving an "unknown user" error, and how do I fix it?"
```

**Comment by iskyOS** [comment]
> So much better, thank you! Caught what I think is a small typo, but otherwise looks great. One thing I noticed: the screenshots still have the "Recipes" tab. We should swap those out once the new release is cut (probably middle of next week).

Thanks, @taylorsperry - let's move forward with merging this PR, then revisit those updated screenshots as soon as we can get them. I just don't want to miss the next release cut.

**Pull request merged**